src/frontend/cli.rs

```diff
@@ -314,176 +407,270 @@ pub fn main() {
+                                );
+                                Ok(deployment)
+                            }
+                            Some(Err(e)) => Err(e),
+                            None => {
+                                let default_deployment_path = get_default_deployment_path(&manifest, &network);
+                                let (deployment, _) = match generate_default_deployment(&manifest, &network) {
+                                    Ok(deployment) => deployment,
+                                    Err(message) => {
+                                        println!("{}", red!(message));
+                                        std::process::exit(1);
+                                    }
+                                };
+                                let res = write_deployment(&deployment, &default_deployment_path, true);
+                                if let Err(message) = res {
+                                    Err(message)
+                                } else {
+                                    let mut relative_path = PathBuf::from("deployments");
+                                    relative_path.push(default_deployment_path.file_name().unwrap());
+                                    println!("{} {}", green!("Generated file"), relative_path.display());
+                                    Ok(deployment)
+                                }
+                            }
+                        }
+                    }
+                    (None, Some(deployment_plan_path)) => {
+                        let deployment_path = get_absolute_deployment_path(&manifest, &deployment_plan_path);
+                        load_deployment(&manifest, &deployment_path)
+                    }
+                    (_, _) => unreachable!()
+                };
+
+                let deployment = match result {
+                    Ok(deployment) => deployment,
+                    Err(e) => {
+                        println!("{}", e);
+                        std::process::exit(1);
+                    }
+                };
+
+                let contracts_ids = deployment
+                    .contracts
+                    .keys()
+                    .map(|contract_id| contract_id.to_string())
+                    .collect::<Vec<_>>();
+                let node_url = deployment.node.clone().unwrap();
 
                 println!(
-                    "Resolving {} and its dependencies...",
-                    fork_contract.contract_id
+                    "The following deployment plan will be applied:\n{}\n\n{}",
+                    deployment.get_synthesis(),
+                    yellow!("Continue [Y/n]?")
                 );
+                let mut buffer = String::new();
+                std::io::stdin().read_line(&mut buffer).unwrap();
+                if !buffer.starts_with("Y") || !buffer.starts_with("y") || !buffer.starts_with("") {
```

Comment:
This doesn't work either. How about:
```suggestion
                if !buffer.starts_with("Y") && !buffer.starts_with("y") && !buffer.starts_with("\n") {
```